helm/templates/statefulset.yaml

```diff
@@ -0,0 +1,110 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ .Release.Name | default "pccs"  }}
+  labels:
+    app.kubernetes.io/name: {{ .Release.Name | default "pccs" }}
+    app.kubernetes.io/version: {{ trimPrefix "v" (index (split ":" .Values.image) 1) }}
+    app.kubernetes.io/managed-by: Helm
```

Comment:
The meaning of those must comply with docs: https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/#labels

Also, use tpl to generate the labels and include (reuse) it in all labels and selector labels places.